platform/common/spinning_zsync

```diff
@@ -19,6 +19,7 @@ else
     ZSYNC_MESSAGE="Computing zsync delta"
 fi
 
+./fbink --clear
 # Small zsync wrapper so we can print its output while it works...
 ./fbink -q -y -7 -pmh "${ZSYNC_MESSAGE} . . ."
```

Comment:
...

`./fbink -q -c -y -7 -pmh "${ZSYNC_MESSAGE} . . ."`

Does the same without an extra refresh ;).